samplers/parser.go

```diff
@@ -72,9 +72,13 @@ func ParseMetricSSF(metric *ssf.SSFSample) (*UDPMetric, error) {
 		return nil, invalidMetricTypeError
 	}
 	h.Write([]byte(ret.Type))
-	ret.Value = float64(metric.Value)
+	if metric.Metric == ssf.SSFSample_SET {
+		ret.Value = metric.Message
+	} else {
+		ret.Value = float64(metric.Value)
+	}
 	ret.SampleRate = metric.SampleRate
-	tempTags := make([]string, len(metric.Tags))
+	tempTags := make([]string, 0)
```

Comment:
I think this is actually a bugfix as well. I ran in to some situations where the incoming packet had no tags and it was causing failure in other places.